get-version.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
```

Comment:
Not certain if you want this at top-level, or put inside release-tools/.  If the latter, we'll need to distribute that directory.